fast-double-parser-8/src/main/java/io/deephaven/csv/tokenization/FastCustomDoubleParser8.java

```diff
@@ -11,25 +11,30 @@
  */
 public final class FastCustomDoubleParser8 implements CustomDoubleParser {
     /**
-     * Wraps the {@link ByteSlice} in a reusable {@link CharSequence} so that it can be passed to
-     * {@link #parse(CharSequence)}
+     * Equivalent to {@link JavaDoubleParser#parseDouble} (with the {@link ByteSlice} interpreted as a
+     * {@link CharSequence}.
      *
      * @param bs The byte slice.
-     * @return The parsed value if successful. Otherwise, throws NumberFormatException.
+     * @return The parsed value if successful. Otherwise, throws {@link NumberFormatException}.
      */
     @Override
     public double parse(ByteSlice bs) throws NumberFormatException {
-        return FastDoubleParser.parseDouble(bs);
+        // FastDoubleParser used to throw NumberFormatException on empty input. Now it throws IllegalArgumentException.
+        // We will make a change request at the FastDoubleParser project.
+        if (bs.size() == 0) {
+            throw new NumberFormatException("empty input");
```

Comment:
Looks like we can bump to 0.5.2 and this gets fixed? https://github.com/wrandelshofer/FastDoubleParser/releases/tag/v0.5.2